node/src/index.ts

```diff
@@ -7,7 +7,7 @@
 export { BlockEventsOptions } from './blockeventsbuilder';
 export { BlockEventsRequest, BlockEventsWithPrivateDataRequest, FilteredBlockEventsRequest } from './blockeventsrequest';
 export { ChaincodeEvent } from './chaincodeevent';
-export { ChaincodeEventsOptions } from './chaincodeeventsbuilder';
+export { Checkpointer } from './checkpointer';
```

Comment:
`CheckpointAsyncIterable` needs to be exported. Also probably:
```typescript
export * as checkpointers from './checkpointers';
```
Similar to `signers` below